core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2264,6 +2264,9 @@ private void loadMetadataInternal(RpcContext rpcContext, LockedInodePath inodePa
             listOptions.setRecursive(false);
           }
           UfsStatus[] children = ufs.listStatus(ufsUri.toString(), listOptions);
+          if (children == null) {
```

Comment:
can you add a comment on when this would happen, and why it's correct to return?

Reply:
will do, and changed it to throw an exception instead.  It can be null for a number of reasons, so I just use a relatively generic IOException.